src/main/resources/application.yml

```diff
@@ -6,7 +6,7 @@ hiu:
   name: ${HIU_NAME}
   dataPushUrl: ${DATA_PUSH_URL}
   consentNotificationUrl: ${CONSENT_NOTIFICATION_URL}
-  valueSets: classpath:/static/ValueSet/valuesets.json
+  valueSets: classpath:/static/ValueSet/ncg-valuesets.json
```

Comment:
should this not be driven by configuration? we have the a default value (am ok with making ref to ndhm defined hiTypes and purposes).